src/Backend/Modules/Blog/Actions/Add.php

```diff
@@ -62,7 +62,7 @@ private function loadForm(): void
         $this->form->addText('title', null, null, 'form-control title', 'form-control danger title');
         $this->form->addEditor('text');
         $this->form->addEditor('introduction');
-        $this->form->addRadiobutton('hidden', $rbtHiddenValues, false);
+        $this->form->addRadiobutton('hidden', $rbtHiddenValues, 0);
```

Comment:
the signature doens't expect a boolean, that value is cast to string, I changed it because false translated to empty string and that caused the code not to work anymore